app-modules/task/src/Notifications/TaskAssignedToUserNotification.php

```diff
@@ -75,13 +79,25 @@ public function toDatabase(object $notifiable): array
     {
         $url = EditTask::getUrl(['record' => $this->task]);
 
+        $student_prospect_url = ! empty($this->task->concern)
+          ? (
+              $this->task->concern instanceof Student
+            ? ViewStudent::getUrl(['record' => $this->task->concern])
+            : ViewProspect::getUrl(['record' => $this->task->concern])
+          )
+          : null;
```

Comment:
Can you please refactor this to be a `match` with arms `$this->task->concern instanceof Student`, `$this->task->concern instanceof Prospect` and then the `default` handles null?

Also please keep variables in `$camelCase`

Reply:
Done.
Sorry for variables name bug.